pkgs/applications/version-management/redmine/gemset.nix

```diff
@@ -124,6 +152,8 @@
   };
   capybara = {
     dependencies = ["addressable" "mini_mime" "nokogiri" "rack" "rack-test" "regexp_parser" "xpath"];
+    groups = ["test"];
```

Comment:
@manveru up until this point I have been using `bundix` from NixOS `18.09` because `bundix` past that point added `groups` to `gemset.nix`. Mostly because I've been lazy I haven't dug into how to fix this...

I'll be forced to use a newer `bundix` now, as we discussed over IRC. This is a good thing... but I'm wondering if you have any insight into how I can fix the current error I'm getting here:

https://logs.nix.ci/?key=nixos/nixpkgs.84823&attempt_id=21b0f328-2508-4399-8376-b725c7c03c3c
```
machine# [   18.110780] flgad10fxwl3fxvcndgc5cd1x4f3iz6j-unit-script-redmine-pre-start[868]: Bundler::GemNotFound: Could not find capybara-3.25.0 in any of the sources
```

Reply:
You should be able to add `test` to the groups in the `default.nix`

Reply:
@manveru thanks for the simple and straight forward answer, though I'm left wondering why `redmine` is choking because of `capybara` when that is supposed to be a test only thing... :thinking: